specification/eventgrid/resource-manager/readme.md

```diff
@@ -0,0 +1,40 @@
+# EventGrid
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Azure EventGrid.
+
+
+
+---
+## Getting Started
+To build the SDK for Azure EventGrid, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for the Azure EventGrid API.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-06-preview
```

Comment:
Looking at the tag field in a few other services, it seems to follow the pattern of "package-<year>-<month>" or "package-<year>-<month>-preview", the current name (package-2017-06-preview) was chosen to be consistent with that naming scheme, so if it doesn't cause any side-effects, would like to continue with the current name.